packages/legend-query-builder/src/components/QueryBuilder.tsx

```diff
@@ -229,127 +215,13 @@ export const QueryBuilder = observer(
   (props: { queryBuilderState: QueryBuilderState }) => {
     const { queryBuilderState } = props;
     const applicationStore = queryBuilderState.applicationStore;
+    const hideQueryBuilderHeader = applicationStore.pluginManager
```

Comment:
as mentioned we should not be making any changes in legend-application. 

Two modules i see changes are in query-builder and application-query. 

Also are we really hiding the header in application query. the header is just a simple panel with Legend Query right?